Majella add networks comparison (#951)
w3f/polkadot-wiki: docs/learn-kusama-vs-polkadot.md

For those who want to validate on Kusama *and* Polkadot, the same setup and infrastructure can be used for both networks.

### Use cases

Polkadot is and always will be the primary network for the deployment of enterprise-level applications and those that entail high-value transactions requiring bank-level security and stability.
The initial use case for Kusama is as a pre-production environment, a “canary network”. Building on Kusama first allows teams to test things out in a live, fully decentralized and community-controlled network with real-world conditions and lower stakes in the event of problems or bugs than on Polkadot.
Many projects will maintain parachains on both networks, experimenting and testing new technologies and features on Kusama before deploying them to Polkadot.
Some teams will decide just to stay on Kusama, which is likely to be a place where we see some exciting experimentation with new technologies going forward.

> Projects that require high-throughput but don’t necessarily require bank-like security, such as some gaming, social networking, and content distribution applications, are particularly good candidates for this use-case.

Reviewer: ```suggestion
Projects that require high-throughput but don’t necessarily require bank-like security, such as some gaming, social networking, and content distribution applications, are particularly good candidates for this use case.
```

Kusama may also prove to be the perfect environment for ambitious experiments with new ideas and new innovations in areas like governance, incentives, monetary policy, and DAOs (decentralized autonomous organizations).

> Future upgrades to the Polkadot runtime will also likely be deployed to Kusama before Polkadot mainnet. This way, not only will we be able to see how these new technologies and features will perform under real-world conditions before bringing them to Polkadot, but teams who have deployed to both networks will also get an advanced look at how their own technology will perform under those upgrades.

Reviewer: ```suggestion
Future upgrades to the Polkadot runtime will also likely be deployed to Kusama before Polkadot mainnet. This way, not only will we be able to see how these new technologies and features will perform under real-world conditions before bringing them to Polkadot, but teams who have deployed to both networks will also get an advance look at how their own technology will perform under those upgrades.
```

## Going forward

Ultimately, Kusama and Polkadot will live on as independent, standalone networks with their own communities, their own governance, and their own complementary use cases, though they will continue to maintain a close relationship, with many teams likely deploying applications to both networks.
In the future, we’re also likely to see Kusama bridged to Polkadot for cross-network interoperability.

> Web3 Foundation and Parity Technologies remain committed to both networks going forward, providing crucial support and guidance to teams building for the ecosystem.

Reviewer: ```suggestion
Web3 Foundation remains committed to both networks going forward, providing crucial support and guidance to teams building for the ecosystem.
```

## Explore more

- [About Kusama](https://kusama.network)
- [The Kusama Wiki](https://guide.kusama.network)

> - [Kusama on polkadot.js](https://kusama.dotapps.io)

Reviewer 2: ```suggestion
- [Kusama on Polkadot-JS Apps](https://kusama.dotapps.io)
```
I'd just change the name of the Polkadot UI here

---
id: learn-kusama-vs-polkadot
title: Comparing Polkadot and Kusama
sidebar_label: Comparing Polkadot and Kusama
---

> # What is the difference between Polkadot and Kusama?

Reviewer 3: This will cause the page to have two header titles since the title is taken from the metadata above. So it would end up rendering like the below. I would recommend changing the `title` field in the metadata and deleting the first header.

# Comparing Polkadot and Kusama

# What is the difference between Polkadot and Kusama?

Although they are built...

Author: Changed

> Although they are built on nearly the same codebase, Polkadot and Kusama are independent, standalone networks with different priorities.  

Reviewer 3: Technically they are built from the _same_ codebase so it may be better to rephrase this as saying they "share many parts of their code".

Author: Changed

Kusama is wild and fast; great for bold experimentation and early-stage deployment. Polkadot is more conservative, prioritizing stability and dependability.

## What the two networks have in common

> Kusama was released as an early version of the same code to be used in Polkadot, which means they share the same underlying architecture: a multichain, heterogeneously-sharded design based on nominated proof of stake (NPoS).

Reviewer 2: ```suggestion
Kusama was released as an early version of the same code to be used in Polkadot, which means they share the same underlying architecture: a multichain, heterogeneously-sharded design based on Nominated Proof of Stake (NPoS).
```
just a nitpick =)

> Both networks also share key innovations like on-chain governance, hot-swappable runtimes for forkless, on-chain upgrades, and cross-chain message passing (XCMP) for interoperability.

Reviewer 2: ```suggestion
Both networks also share key innovations like on-chain governance, hot-swappable runtimes for forkless, on-chain upgrades, and Cross-Chain Message Passing (XCMP) for interoperability.
```

Governance on both Polkadot and Kusama is designed to be decentralized and permissionless, giving a say in how the network is run to everyone who owns the native token (DOT for Polkadot, KSM for Kusama). Therefore, **over time the networks will evolve independently, converging or diverging according to the decisions of their respective communities.**

## Key differences

There are a few important distinctions to be made.

> ![polkadot_vs_kusama](/img/polkadot_vs_kusama.png)

Reviewer 2: I would recommend here to place the image inside the `assets` folder. Logan's new img optimizer will be optimizing the images inside that folder.
